src/test/java/org/opensearch/ad/rest/SecureADRestIT.java

```diff
@@ -378,4 +387,46 @@ public void testPreviewAnomalyDetectorWithNoReadPermissionOfIndex() throws IOExc
         );
         Assert.assertTrue(exception.getMessage().contains("no permissions for [indices:data/read/search]"));
     }
+
+    public void testValidateAnomalyDetectorWithWriteAccess() throws IOException {
+        // User Alice has AD full access, should be able to validate a detector
+        AnomalyDetector aliceDetector = createRandomAnomalyDetector(false, false, aliceClient);
+        Response validateResponse = validateAnomalyDetector(aliceDetector, aliceClient);
+        Assert.assertNotNull("User alice validatedDetector", validateResponse);
```

Comment:
Should the response be more user-friendly, like `validated detector` instead of `validatedDetector`?